### PR TITLE
Added skip button in new jump

### LINKED ISSUE #192: Add skipped jump
On New form add button or `link_to` "Skip" in the same fashion as on jump page. It should create skipped jump in given category

### PR DESCRIPTION
**Pull request by Konfeta3-33**
fixes #192

### CODE REVIEW

**Review by skyksandr — APPROVED** [review]
Looks good. Two improvements below

**Comment by Konfeta3-33** [comment]
fixes #192

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -14,4 +14,9 @@ def set_current_attributes
 
     Current.user = current_user
   end
+
+  rescue_from Pundit::NotAuthorizedError do
+    flash[:alert] = I18n.t('general.not_allowed_operation')
+    render file: 'public/404.html', status: :forbidden
```
Comment (skyksandr) [inline app/controllers/application_controller.rb:20]: Any reason you render `404` instead of `422`

**Review by skyksandr — APPROVED** [review]
Overall good

**Pull request merged**